R/codec_tdr.R

```diff
@@ -84,7 +84,10 @@ codec_data <- function(name, geography = cincy::tract_tigris_2010, geometry = FA
     d_sf <- dplyr::left_join(d, cincy::tract_tigris_2010, by = "census_tract_id_2010")
     d_sf <- sf::st_as_sf(d_sf)
     d_int <- cincy::interpolate(from = d_sf, to = geography, weights = "pop")
-    if (!geometry) d_int <- sf::st_drop_geometry(tibble::as_tibble(d_int))
+    if (!geometry) {
+      d_int <- sf::st_drop_geometry(tibble::as_tibble(d_int)) |>
```

Comment:
I think the `sf::st_drop_geometry()` may be unnecessary here.  I was playing around with this in another repository and if we use `as_tibble`, then `st_drop_geometry` won't do anything because it is not of class `sf`.  Once it is a tibble, then the geometry column isn't "sticky" and it will drop with `select()`.